lms/djangoapps/certificates/tests/test_signals.py

```diff
@@ -251,6 +251,7 @@ def test_cert_generation_on_photo_verification_self_paced(self):
                 mock_generate_certificate_apply_async.assert_called_with(kwargs={
                     'student': unicode(self.user_one.id),
                     'course_key': unicode(self.course_one.id),
+                    'expected_verification_status': unicode('approved')
```

Comment:
It would perhaps be better to refer to this as `SoftwareSecurePhotoVerification.STATUS.approved`.